### PR TITLE
From stream to Hbase

### PR DESCRIPTION
**Pull request by HilmiYildirim**
New Pull Request for Pull Request https://github.com/apache/flink/pull/702

### CODE REVIEW

File: flink-staging/flink-hbase/src/test/java/org/apache/flink/addons/hbase/example/HBaseWriteStreamExample.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+			public String next() throws Exception {
+				return 	String.valueOf(Math.floor(Math.random() * 100));
+			}
+
+		});
+		dataStream.write(new HBaseOutputFormat(), 0L);
+
+		try {
+			env.execute();
+		} catch (Exception e) {
```
Comment (rmetzger) [inline flink-staging/flink-hbase/src/test/java/org/apache/flink/addons/hbase/example/HBaseWriteStreamExample.java]: the try / catch block here is not really necessary. You can also make the main method throw the exception.

(You don't need to update the PR just for that)

**Comment by rmetzger** [comment]
+1 to merge the change

File: flink-staging/flink-hbase/src/test/java/org/apache/flink/addons/hbase/example/HBaseWriteStreamExample.java (hunk 1 above)
Comment (HilmiYildirim) [inline flink-staging/flink-hbase/src/test/java/org/apache/flink/addons/hbase/example/HBaseWriteStreamExample.java]: Great

**Comment by StephanEwen** [comment]
Will merge this...

**Pull request closed without merging**